README.md

```diff
@@ -18,8 +20,8 @@ To get a local copy up and running follow these simple example steps.
 ## Author
 👤 Donald Akite
 
-- GitHub: @quavo19
-- Twitter: @donaldakite
+- GitHub: @ https://github.com/quavo19
+- Twitter:@ https://twitter.com/DonaldAkite
 
 ## Feel free to check the issues page.
```

Comment:
[OPTIONAL] Please make sure you add the contributing section. You can change `## Feel free to check the issues page.` to the section below.
  ```
  ## 🤝 Contributing
  
  Contributions, issues, and feature requests are welcome!
  
  Feel free to check the [issues page](https://github.com/Owusu-Desmond/blog-app/issues).
  
  ```